internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -60,7 +60,8 @@ func New(vmName string, sshUser string, sshPassword string, cpu uint32, memory u
 }
 
 func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err error) {
-	vm, err := NewVMClonedFrom(ctx, tart.vmName, tart.cpu, tart.memory, config.TartOptions.LazyPull, config.Logger())
+	tmpVmName := fmt.Sprintf("cirrus-cli-%d", config.TaskID)
```

Comment:
Do you think it's worth to keep the UUID at the end? Just in case the task ID uniqueness assumption holds false in tests or somewhere else.

Reply:
Don't think so but it's fine by me. See [81af546](https://github.com/cirruslabs/cirrus-cli/pull/572/commits/81af546c071717e402ee3f039e8514a23219108e)